src/DataTable/DataTable.js

```diff
@@ -312,6 +313,16 @@ const DataTable = memo(({
             />
           )}
 
+          {enabledPagination && paginationPosition === 'top' && (
+            <Pagination
```

Comment:
Rather than repeat the pagination code it might be better to have this as a separate code function that returns the pagination component so {enabledPagination && paginationPosition === 'top' && renderPagination()};

Reply:
good idea, done!